### PR TITLE
Published Properties Refactor

### PR DESCRIPTION
**Pull request by GordonSmith**
Refactor published properties to use set/get methods.

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@mlzummo Please Review

File: src/c3chart/Gauge.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
             context.click(clickEvent, d.id);
         };
         this._config.data.color = function (color, d) {
-            return context._palette(context._data, context._low, context._high);
+            return context._palette(context._data, context.low(), context.hight());
```
Comment (mlzummo) [inline src/c3chart/Gauge.js]: hight* -> high

File: src/google/Line.js [hunk 2]
```diff
@@ -100,119 +100,123 @@
     Line.prototype.getChartOptions = function () {
         var retVal = CommonND.prototype.getChartOptions.apply(this, arguments);
 
-        retVal.lineWidth = this._lineWidth;
-        retVal.lineDashStyle = this._gloabLineDashStyle;
-        retVal.orientation = this._orientation;
-        retVal.pointShape = this._globalPointShape;
-        retVal.pointSize = this._globalPointSize;
+        retVal.lineWidth = this.lineWidth();
+        retVal.lineDashStyle = this.gloabLineDashStyle();
```
Comment (mlzummo) [inline src/google/Line.js]: typo

File: src/google/Line.js [hunk 3]
```diff
@@ -100,119 +100,123 @@
...
+        retVal.hAxis.minValue = this.hAxisMinValue();
+        retVal.hAxis.maxValue = this.hAxisMaxValue();
         
-        retVal.hAxis.hAxisAllowContainerBoundaryTextCufoff = this._hAxisAllowContainerBoundaryTextCufoff;
-        retVal.hAxis.slantedText = this._hAxisSlantedText;
-        retVal.hAxis.slantedTextAngle = this._hAxisSlantedTextAngle;
-        retVal.hAxis.maxAlternation = this._hAxisMaxAlternation;
-        retVal.hAxis.maxTextLines = this._hAxisMaxTextLines;
-        retVal.hAxis.minTextSpacing = this._hAxisMinTextSpacing;
+        retVal.hAxis.hAxisAllowContainerBoundaryTextCufoff = this.hAxisAllowContainerBoundaryTextCufoff();
```
Comment (mlzummo) [inline src/google/Line.js]: its suppose to be spelled Cutoff .. the publish param might be spelled wrong too

**Comment by GordonSmith** [comment]
Changes made as per comments:  
http://rawgit.com/GordonSmith/Visualization/PERSIST_DEFAULT_VALUES/demos/dermatology.html
http://rawgit.com/GordonSmith/Visualization/PERSIST_DEFAULT_VALUES/demos/test.html

**Comment by mlzummo** [comment]
looks good

**Pull request merged**